docs/wip/how-to-rollback.md

```diff
@@ -1,12 +1,115 @@
-## Step by step on how to rollback?
+## ⚠️ Warning: this will only work when we start using VTEX infrastructure to build the images. Is also possible to build with Netlify[] — for rollback in Netlify, you can use it's tools.
```

Comment:
```suggestion
## ⚠️ Warning: this will only work when we start using VTEX infrastructure to build the images. Is also possible to build with Netlify[] — for rollback in Netlify, you can use its tools.
```